timeseries/src/autogluon/timeseries/models/chronos/model.py

```diff
@@ -230,7 +230,7 @@ def load_model_pipeline(self, context_length: Optional[int] = None):
                 "`import torch; torch.cuda.is_available()` returns `True`."
             )
 
-        device = self.device or ("cuda" if gpu_available else "auto")
+        device = self.device or ("cuda" if gpu_available else "cpu")
```

Comment:
Why do we remove auto ?

Reply:
This is being cherry-picked from https://github.com/autogluon/autogluon/pull/4565. Timeseries tests were failing on MacOS due to MPS memory overflow.

Reply:
The timeout may mean that the tests may be getting stuck

Reply:
MutiGPU tests passed when switching back from `uv pip install` to `pip install`.